icu4c/source/test/fuzzer/fuzzer_driver.cpp

```diff
@@ -10,11 +12,49 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size);
 
 int main(int argc, char* argv[])
 {
-  (void) argc;
-  (void) argv;
-  const char *fuzzer_data = "abc123";
-   
-  LLVMFuzzerTestOneInput((const uint8_t *) fuzzer_data, strlen(fuzzer_data));
+  bool show_warning = true;
```

Comment:
Shouldn't we use ICU coding conventions here? Indent by 4, usually put the `{` at the end of the decl head, ...